### DISCUSSION TITLE
Way to check for types of afflictions (specifically types and not identifiers) in conditionals?

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#12979] [Code questions and modding support]
Title. I don't know of any mods I can learn from that do it nor can I find it in documentation. Is there even a way to do this?

**Comment by G-M-twostay** [comment]
the conditional checks for both affliction identifier as well as its type, so just use the type name as you would with identifier.

**Reply by ghost** [reply]
thanks, will try it
